Resend Verification Email Can Be Abused to Spam Mail Server #35965 (#35971)
RocketChat/Rocket.Chat: .changeset/eighty-windows-join.md

---
"@rocket.chat/meteor": patch
---

> fixes and issue where the Resend Verification Email Can Be Abused to Spam Mail Server  

Reviewer: ```suggestion
Fixes an issue where the Resend Verification Email could be abused to spam mail servers
```
